visa/GraphColor.cpp

```diff
@@ -4849,8 +4855,8 @@ void Augmentation::addSIMDIntfForRetDclares(G4_Declare* newDcl)
     if (dclIt == retDeclares.end())
     {
         MaskDeclares newMask;
-        newMask.first.resize(liveAnalysis.getNumSelectedGlobalVar());
-        newMask.second.resize(liveAnalysis.getNumSelectedGlobalVar());
+        newMask.first.reserve(liveAnalysis.getNumSelectedGlobalVar()/32);
```

Comment:
why divide by 32?

Reply:
This was an ad-hoc decision to keep the memory footprint the same as before. We could try not calling reserve() at all and letting it resize on the fly as necessary, but that may come back with a performance penalty again.